experimental/Schemes/elliptic_surface.jl

```diff
@@ -511,18 +515,22 @@ function relatively_minimal_model(E::EllipticSurface)
     Y0 = Y1
     inc_Y0 = inc_Y1
     X0 = X1
+    set_attribute!(Y0, :is_irreducible, true)
+    set_attribute!(Y0, :is_reduced=>true)
```

Comment:
I am impartial to which syntax variant you use for setting attributes, but it is a bit jarring that in this code block it alternates between the two repeatedly ;-).

Feel free to leave it and ignore me :-)

Reply:
Agreed.